File: CRS-LT-POS-Development/src/com/lt/bean/Professor.java
package com.lt.bean;

public class Professor {

	private String ProfessorName;
	private String ProfessorPassword;
	private Long ProfessorContactNo;
	private String ProfessorEmailId;
	private String ProfessorAddress;
	
	
	public Professor(String name,String password, Long contact, String email, String address)
	{
		this.ProfessorName=name;
		this.ProfessorPassword=password;
		this.ProfessorContactNo=contact;
		this.ProfessorEmailId=email;
		this.ProfessorAddress=address;		
	}
	
	public String getProfessorName() {
		return ProfessorName;
	}
	public void setProfessorName(String professorName) {
		ProfessorName = professorName;
	}
	public Long getProfessorContactNo() {
		return ProfessorContactNo;
	}
	public void setProfessorContactNo(Long professorContactNo) {
		ProfessorContactNo = professorContactNo;
	}
	public String getProfessorEmailId() {
		return ProfessorEmailId;
	}
	public void setProfessorEmailId(String professorEmailId) {
		ProfessorEmailId = professorEmailId;
	}
	public String getProfessorAddress() {
		return ProfessorAddress;
	}
	public void setProfessorAddress(String professorAddress) {
		ProfessorAddress = professorAddress;
	}
	public String getProfessorPassword() {
		return ProfessorPassword;
	}
	public void setProfessorPassword(String professorPassword) {
		ProfessorPassword = professorPassword;
	}	
}

File: CRS-LT-UI-DEVELOPMENT/src/model/professor.ts
/**
 * 
 * @author G4-FullStackGroup
 * Professor Class
 * 
 */
export class Professor {

	public  professorName:string;
    public userName:string;
    public password:string;
	public  department:string;
	public  professorAddress:string;
	
    /**
	 *Parameterized Constructor
	 * @return values
	 */
    constructor(professorName:string, userName:string, password:string, department:string, 
        professorAddress:string)
    {
    
        this.professorName=professorName;
        this.userName=userName;
        this.password=password;
        this.department=department;
        this.professorAddress=professorAddress;
      

    }
}